src/main/java/teamroots/embers/tileentity/TileEntityEmberBore.java

```diff
@@ -380,9 +381,8 @@ public ItemStack insertItemInternal(int slot, ItemStack stack, boolean simulate)
 
         @Override
         public ItemStack extractItem(int slot, int amount, boolean simulate) {
-            ItemStack currentFuel = super.extractItem(slot, amount, true);
-            int burntime = TileEntityFurnace.getItemBurnTime(currentFuel);
-            if (slot == SLOT_FUEL && burntime != 0) {
+            //disallow extraction from fuel slot
```

Comment:
You can't do this, if a lava bucket is consumed, the empty bucket stays in the fuel slot, and obviously must be removable.

Reply:
Forgot about that part. The new code assumes that container items have a count of 1, which should be the case unless someone overrides `getContainerItem(ItemStack)`. I think it's worth the performance to not account for stacked container items (I've never seen any anyway).